### ISSUE TITLE
Support PARTITIONED BY with procedure call in it

### ISSUE THREAD

**Issue by looi** [xnuinside/simple-ddl-parser#154]
Use case is Spark DDL with Iceberg, example:

```sql
create table a (b timestamp, c int)
using iceberg
partitioned by (months(b))
location 's3://tables/a'
```

More examples at https://iceberg.apache.org/docs/latest/spark-ddl/#partitioned-by

**Comment by xnuinside** [comment]
@looi hi, thanks for reporting the issue, I will add support

**Comment by xnuinside** [comment]
fix added in https://github.com/xnuinside/simple-ddl-parser/issues/154

**Comment by xnuinside** [comment]
fix released in version 0.27.0, if will be anything else - feel free to open new issue, and thanks for reporting @looi !

**Comment by looi** [comment]
Awesome!
